staging/src/k8s.io/apiserver/plugin/pkg/audit/truncate/truncate.go

```diff
@@ -128,12 +128,11 @@ func truncate(e *auditinternal.Event) *auditinternal.Event {
 }
 
 func (b *backend) Run(stopCh <-chan struct{}) error {
-	// Nothing to do here
-	return nil
+	return b.delegateBackend.Run(stopCh)
```

Comment:
can we add that to the (not existing) godoc? Either here or above for the constructor at least: It runs and shuts down the delegate.

Reply:
I have added comment both to the truncate and buffered backends.